lxd/storage/utils.go

```diff
@@ -737,12 +760,17 @@ func VolumeUsedByRunningInstancesWithProfilesGet(s *state.State, projectName str
 			// "container////bla" but only against "container/bla".
 			cleanSource := filepath.Clean(dev["source"])
 			if cleanSource == volumeName || cleanSource == volumeNameWithType {
-				instUsingVolume = append(instUsingVolume, inst.Name())
+				err = instanceFunc(inst, p, profiles)
+				if err != nil {
+					return err
+				}
 			}
 		}
-	}
 
-	return instUsingVolume, nil
+		return nil
+	})
+
+	return err
```

Comment:
Hard to see in diff but would it have looked too weird to `return` the entire function call instead?

Reply:
@stgraber yep that would be clearer to return the function's return value directly, will change.